openlibrary/templates/books/custom_carousel.html

```diff
@@ -71,7 +71,14 @@ <h2 class="home-h2"><a href="$url">$title</a></h2>
         </div>
         <script type="text/javascript">
               window.q.push(function() {
-                Carousel.add('.carousel-$key', 6, 5, 4, 3, 2, 1, '$key', '$load_more_url');
+                Carousel.add(
+                  '.carousel-$key', // dom element to bind slick
+                  6, 5, 4, 3, 2, 1, { // num books per responsive breakpoint
+                  $if load_more:
+                     url: '$:(load_more["url"].replace("&amp;", "&"))',
```

Comment:
This could really do with more explanation. Why is &amp the only character there, and why is it being HTML escaped here in the first place?